src/app/src/main/kotlin/org/vaccineimpact/api/app/models/CreateUser.kt

```diff
@@ -1,7 +1,7 @@
 package org.vaccineimpact.api.app.models
 
+import org.vaccineimpact.api.models.validation.AllowedFormat
 import org.vaccineimpact.api.security.BasicUserProperties
-import org.vaccineimpact.api.serialization.validation.AllowedFormat
```

Comment:
I've moved the validation annotations into the submodule so that we can use them on models.